build.sbt

```diff
@@ -181,140 +219,127 @@ wartremoverErrors in (Compile, compile) ++= (Warts.unsafe.toSet -- Set(Wart.Any,
                                                                        Wart.DefaultArguments,
                                                                        Wart.Throw)).toSeq
 lazy val scalafixSettings =
-  Seq(addCompilerPlugin(scalafixSemanticdb),
-      scalacOptions ++= Seq("-Yrangepos", "-Ywarn-unused-import", "-P:semanticdb:failures:warning"))
+  Seq(addCompilerPlugin(scalafixSemanticdb), scalacOptions ++= Seq("-Yrangepos", "-P:semanticdb:failures:warning"))
 
 lazy val core = (project in file("core"))
+  .enablePlugins(MacrosCompiler)
   .settings(scalafixSettings)
   .settings(
     name := "jsonapi-scala-core",
-    scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12", "2.12.10"),
+    scalaVersion := "2.13.3",
+    crossScalaVersions := Seq("2.11.12", "2.12.10", "2.13.3"),
     scalacOptions ++= {
       if (scalaVersion.value startsWith "2.11.") {
         scala211
-      } else {
+      } else if (scalaVersion.value startsWith "2.12.") {
         scala212
-      }
+      } else
+        scala213
     },
     libraryDependencies ++= Seq(
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-      compilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
+      "org.scala-lang"   % "scala-reflect"    % scalaVersion.value,
       "com.chuusai"      %% "shapeless"       % "2.3.3",
-      "com.qvantel"      %% "scala-inflector" % "1.3.6",
+      "com.qvantel"      %% "scala-inflector" % "1.4.0",
       "io.spray"         %% "spray-json"      % "1.3.5",
-      "io.lemonlabs"     %% "scala-uri"       % "0.5.7",
+      "io.lemonlabs"     %% "scala-uri"       % "1.4.10",
       "net.virtual-void" %% "json-lenses"     % "0.6.2" excludeAll
-        ExclusionRule(organization = "org.parboiled", name = "parboiled-scala_2.11"),
-      "org.typelevel" %% "cats-effect" % "1.4.0",
-      "co.fs2"        %% "fs2-io"      % "1.0.5"
-    ) ++ testDeps
+        ExclusionRule(organization = "org.parboiled", name = "parboiled-scala_2.11")
+    ) ++ (if (scalaVersion.value startsWith "2.11.")
```

Comment:
Not sure which versions to use here, should we upgrade for all versions or only Scala 2.13?